### PR TITLE
pre-assesment gc-freatures

### PR DESCRIPTION
**Pull request by ipaksc**

### CODE REVIEW

File: components/gc-features/reports/a11y-1-fr.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"language": "fr",
+	"description": "Partielle. Pré-évaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
+	"tag": "gc-features",
+	"parentdir": "gc-features",
+	"parentPage": "Promotions contextuelles",
+	"parentPageURL": "gc-features",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-04",
+	"layout": "assessment_wrote_en-fr",
+	"reportURL": "a11y-1.json"
```
Comment (duboisp) [inline components/gc-features/reports/a11y-1-fr.html:13]: The name need to be adjusted.

```suggestion
	"reportURL": "pre-a11y-1.json"
```

**Review by duboisp — CHANGES_REQUESTED** [review]
I reviewed and tested locally.

Other than the little change, everything look good once that change is completed.

File: components/gc-features/reports/a11y-1-fr.html (hunk 1 above)
Comment (ipaksc) [reply components/gc-features/reports/a11y-1-fr.html]: Fixed

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally. Work as expected.

Thanks @ipaksc for the pre-assessment and the assessment

**Pull request merged**